lib/Sensors/Gateway.js

```diff
@@ -14,7 +14,14 @@ function Gateway(sid, ip, hub, model) {
     this.illumination = null;
     this.protoVersion = '';
     this.acPower      = undefined;
-    this.acState      = undefined;
+	// New states for acpartner-----------------
+    this.OnOffCfg     = undefined;		
```

Comment:
Please use only low case names like onOffCfg. Capital letters used in classes.

Reply:
> Please use only low case names like onOffCfg. Capital letters used in classes.

ok! Corrected.

Reply:
ok! Corrected.